modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp

```diff
@@ -546,7 +546,14 @@ renderResponse.setTitle(headerTitle);
 
 						<aui:input dateTogglerCheckboxLabel="never-expire" disabled="<%= dlEditFileEntryDisplayContext.isNeverExpire() %>" name="expirationDate" wrapperCssClass="expiration-date mb-3" />
```

Comment:
Hi @boton , @carloslancha 

I have a doubt here.
First of all the hour is 2 hours on the past now is 05:15PM). But the user has another timezone so when I try to save the expiration date on the past (for the user) it throws an exception (good boooooy) but I want that this has the same timezone than the user.
![image](https://user-images.githubusercontent.com/47524751/119004588-fca37f00-b98e-11eb-9a0a-7a11fe8d2907.png)
And another thing is that I want that the pre-loaded date is 1 year on the future.


but I do not know to make it,


Thanks!

Reply:
The part of the preloaded data is made on 6d1dab547926b6bc16df83a1b0cacabfd1144aa6

![](https://media1.giphy.com/media/26u4lOMA8JKSnL9Uk/giphy.gif)

Reply:
So... this still need to be reviewed?

Reply:
well, the issue about the timezone of the user and the timezone of the web still happens, But I think that is an issue that affects the component (I think that what I understood @boton ) and this pr can pass without this particular.

Reply:
Yes, In my opinion, It' is better to save the review and expiration dates with the same logic that we use in Web Content, and if we need to change in both places.